### PR TITLE
Improve Exception/Connectivity handling with Cluster Tab

### PR DESCRIPTION
**Pull request by lbwexler**
Cluster Tab
* DRY for exception/visibility handling in cluster tab
* Standardize exception handling for log display
* Preserve visual state during cluster restart, instance swap

General
* Improve Error Handling  with mode `toast` to avoid extra click

See related Hoist-Core PR

Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by lbwexler** [comment]
@amcclain -- was trying to get Greg's buy-in here, before asking for your review, but think this change represents my best thinking on this.

**Comment by ghsolomon** [comment]
Just reviewed. This looks great to me - really nice use of `LoadSupport.lastLoadException`, which I'll have to start using!

File: admin/tabs/cluster/ClusterTab.ts [hunk 1]
```diff
@@ -31,19 +28,39 @@ export const clusterTab = hoistCmp.factory({
...
-                          hspacer(25),
-                          tabSwitcher()
-                      ],
-                      flex: 1,
-                      item: tabContainer()
-                  })
-                : placeholder(Icon.server(), 'Select a running instance above.'),
-            mask({bind: model.loadModel})
+            detailPanel(),
+            mask({
```
Comment (ghsolomon) [inline admin/tabs/cluster/ClusterTab.ts:32]: Could split the mask out into its own component if we want to minimize re-renders of the top-level `clusterTab` when `lastLoadException` changes

**Comment by lbwexler** [comment]
Reviewed with ATM as well and will merge this shortly, as we all seem to like the overall approach

We discussed future iterations providing even more information in the mask about the state of the last load attempt.  Useful for reassuring anxious XH developers.

**Pull request merged**